tools/hardhat-viem-example/utils/clients.js

```diff
@@ -0,0 +1,15 @@
+const { http, createPublicClient, createWalletClient} = require('viem');
```

Comment:
where is this file used? if it's used implicitly by hardhat, then maybe add comment here.